nautobot/docs/plugins/development.md

```diff
@@ -59,7 +59,9 @@ The plugin source directory contains all of the actual Python code and other res
 
 ### Create setup.py
 
-`setup.py` is the [setup script](https://docs.python.org/3.6/distutils/setupscript.html) we'll use to install our plugin once it's finished. The primary function of this script is to call the setuptools library's `setup()` function to create a Python distribution package. We can pass a number of keyword arguments to inform the package creation as well as to provide metadata about the plugin. An example `setup.py` is below:
+The`setup.py` script is the [setup script](https://docs.python.org/3.6/distutils/setupscript.html) we'll use to install our plugin once it's finished. The primary function of this script is to call the setuptools library's `setup()` function to create a Python distribution package. We can pass a number of keyword arguments to inform the package creation as well as to provide metadata about the plugin. 
```

Comment:
I imagine we'll want to revise this section at some point to use Poetry and `pyproject.toml` instead of `setup.py`, since all of our "official" plugins use those.

Reply:
Yes, agreed. I just left it alone for now until we're ready to do that. It's on my list.

Reply:
I'm going to do the plugin bootstrap docs for setup.py -> pyproject.toml in another PR.